File: Lista01/GO/ex01.go
package main

import "fmt"

// Definindo a struct Carro
type Carro struct {
    Marca  string
    Modelo string
    Ano    int
}

// Função para exibir os detalhes do carro
func (c Carro) ExibirDetalhes() {
    fmt.Printf("Marca: %s, Modelo: %s, Ano: %d\n", c.Marca, c.Modelo, c.Ano)
}

func main() {
    // Instanciando três carros
    carro1 := Carro{"Fiat", "Pulse", 2022}
    carro2 := Carro{"Chevrolet", "Onix", 2021}
    carro3 := Carro{"Volkswagen", "Polo", 2020}

    // Exibindo os detalhes de cada carro
    carro1.ExibirDetalhes()
    carro2.ExibirDetalhes()
    carro3.ExibirDetalhes()
}

File: Lista01/PY/ex01.py
# Definindo a classe Carro
class Carro:
    def __init__(self, marca, modelo, ano):
        self.marca = marca
        self.modelo = modelo
        self.ano = ano

    # Método para exibir os detalhes do carro
    def exibir_detalhes(self):
        print(f"Carro: {self.marca} {self.modelo}, Ano: {self.ano}")

# Instanciando três objetos da classe Carro
carro1 = Carro("Suzuki", "Jimny", 2022)
carro2 = Carro("Honda", "Civic", 2019)
carro3 = Carro("Fiat", "Pulse", 2021)

# Exibindo os detalhes de cada carro
carro1.exibir_detalhes()
carro2.exibir_detalhes()
carro3.exibir_detalhes()
